### ISSUE TITLE
Temporary spritesheets aren't intercepted by content API

### ISSUE THREAD

**Issue by catamari** [Pathoschild/SMAPI#413]
The `Characters\Farmer\farmer_base.png` asset isn't being intercepted by the content API. I both logged everything passed to my mod's `CanEdit` as well as everything passed to `LoadImpl` in SMAPI itself and that asset was never found.

**Comment by Pathoschild** [comment]
@tstaples That's because the player has a separate content manager constructed via `CreateTemporary()`, which SMAPI can't currently intercept:
```cs
public Farmer()
{
    this.farmerTextureManager = Game1.content.CreateTemporary();
    this.farmerRenderer = new FarmerRenderer(this.farmerTextureManager.Load<Texture2D>("Characters\\Farmer\\farmer_" + (this.isMale ? "" : "girl_") + "base"));
    // [...]
}
```

That's something I'm hoping to address with Chucklefish during the SDV 1.3 alpha, so I'll put this ticket in the backlog for now.

**Comment by Pathoschild** [comment]
Fixed in the upcoming SMAPI 2.6 for Stardew Valley 1.3.
